client/public/index.html

```diff
@@ -2,19 +2,17 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="https://cdn3.iconfinder.com/data/icons/essentials-volume-i/128/ticket-stub-3-512.png" />
```

Comment:
Better to host your own images and icons then linking to an external resource.

Reply:
Fixed.